packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -167,6 +168,26 @@ async function main(): Promise<number | void> {
     }
 
     jestCli = jestCli.withEnv({ ONLY_TEST_PROVIDER_ADAPTERS: adapterProviders.join(',') })
+
+    // Start wrangler dev server with the `wrangler-proxy` for D1 tests
+    if (adapterProviders.includes(AdapterProviders.JS_D1)) {
+      wranglerProcess = execa(
+        'pnpm',
+        [
+          'wrangler',
+          'dev',
+          '--config=./tests/functional/_utils/wrangler.toml',
+          'node_modules/wrangler-proxy/dist/worker.js',
+        ],
+        {
+          preferLocal: true,
+          // stdio: 'inherit',
+          env: {
+            DEBUG: process.env.DEBUG,
+          },
```

Comment:
Is it still needed?

Reply:
You mean `DEBUG`? It won't do anything currently but I think it's ok to keep for later